### PR TITLE
Adding several components

### PR DESCRIPTION
**Pull request by Ahrakiel**
- User
- login
- SignUp
- Connection
- Logo pioupiouTrack

### CODE REVIEW

File: TODO.md [hunk 1]
```diff
@@ -10,3 +10,8 @@
     <ion-menu-button menu="menuAppId"></ion-menu-button>
   </ion-button>
 ```
+
+
+## Christopher notes
```
Comment (ymohl-cl) [inline TODO.md]: please can you remove this useless comment ?

File: src/router/index.ts [hunk 2]
```diff
@@ -24,12 +45,25 @@ const routes: Array<RouteRecordRaw> = [
...
-	}
+	},
 ]
 
 const router = createRouter({
 	history: createWebHistory(process.env.BASE_URL),
 	routes
 })
 
+router.beforeEach((to, from, next) => {
```
Comment (ymohl-cl) [inline src/router/index.ts:56]: oh yeah great implementation !

File: src/scripts/Connection.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export function connection(mail: string, password: string): boolean {
+        if (mail != "christopherbatrier@gmail.com" || password != "password") {
```
Comment (ymohl-cl) [inline src/scripts/Connection.ts:2]: seriously ? you write you password ? :)

File: src/scripts/SignUp.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { User } from "@/store/user";
+
+export function isSimilare(fstPwd: string, scdPwd: string): boolean {
+    if (fstPwd !== scdPwd) {
+        return false;
+    }
+    return true;
+}
+
+// export function isExist(email: string): boolean {
```
Comment (ymohl-cl) [inline src/scripts/SignUp.ts]: remove comment function if you not used

File: src/router/index.ts (hunk 2 above)
Comment (Ahrakiel) [reply src/router/index.ts:56]: ty

File: src/scripts/SignUp.ts (hunk 4 above)
Comment (Ahrakiel) [reply src/scripts/SignUp.ts]: Ok I did it !

File: TODO.md (hunk 1 above)
Comment (Ahrakiel) [reply TODO.md]: Done !

File: src/scripts/Connection.ts (hunk 3 above)
Comment (Ahrakiel) [reply src/scripts/Connection.ts:2]: No one except me might access to the application !! :)

Comment (ymohl-cl) [reply src/scripts/Connection.ts:2]: ok

**Pull request merged**